### DISCUSSION TITLE
New open standard audio plugin: CLAP

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#962] [Ideas]
There is a new open standard audio plugin CLAP (CLever Audio Plug-in API). 
"Bitwig and u-he are excited to announce CLAP (CLever Audio Plug-in API), the new open standard for audio plug-ins and hosts. CLAP offers modern features, innate stability, and rapid support for plug-in and host developers. And since it's open source and liberally licensed, CLAP is a safe bet for the future."
https://u-he.com/community/clap/

Maybe we have this new plugin added to Audiogrider in the near future.
CLAP JUCE Extensions project: https://github.com/free-audio/clap-juce-extensions/

**Comment by emack999** [comment]
That's one more plugin to support, which can slow progress elsewhere.
There's also the relevant question:  What can't a well-written VST3 plugin do already?

**Reply by star-buck** [reply]
Clap is an open format and doesnt need a corporate license agreement that can be revoked in the future by a single company (e.g. like VST by Steinberg, see VST2).

**Reply by hschieck** [reply]
Available CLAP Plugins often are more "native" on a Linux Host, while most of the time VST plugins on Linux need WINE to emulate the Windows API or won't run at all because of Spyware-like Kernel-level Copy Protection. I ditched Cubase in favour of Bitwig, but I use AG to keep my Arturia V Collection, my Eventide Plugins and a bunch of other stuff that won't run natively on Linux. The more applications supporting CLAP, the higher the chance that big players on the market decide to support Linux as a host operating system as well. Probably... Perhaps... I hope. *sigh*
